packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -165,8 +165,7 @@ export type InternalRequestParams = {
    */
   jsModelName?: string
   callsite?: CallSite
-  /** Headers metadata that will be passed to the Engine */
-  headers?: Record<string, string> // TODO what is this
```

Comment:
This is used by the Accelerate team, we'll need to merge other work to provide an alternative

Reply:
Restored the property and functionality for now

Reply:
One note: once #17356 is merged, we could remove it from our side, right?

Reply:
Yes